### DISCUSSION TITLE
Character notes

### DISCUSSION THREAD

**Discussion by Odoamar** [fmasa/wfrp-master#105] [Ideas]
![Screenshot_2023-05-05-16-07-43-466_cz frantisekmasa dnd](https://user-images.githubusercontent.com/77120529/236524550-d80a8e10-cb5d-4281-b830-6fb3cd4c79ca.jpg)
There is my idea about "Notes" tab for characters. 

I think all text boxes should be more consistent. Now "Additional notes" are editable only via edit button on top. I prefer the solution implemented in "Character ambitions" 

Motivations are not visible - I assume this is a bug?

Proposed boxes: 

- GM only notes: https://github.com/fmasa/wfrp-master/issues/35
- Shared info: https://github.com/fmasa/wfrp-master/discussions/106

**Comment by fmasa** [comment]
> I think all text boxes should be more consistent. Now "Additional notes" are editable only via edit button on top. I prefer the solution implemented in "Character ambitions"

This came up several times and I agree the current UI is confusing, so I am all for aligning it with how Ambitions card works! :+1: 

> Motivations are not visible - I assume this is a bug?

Oversight really. Yes, this should be here :slightly_smiling_face: 

---

All in all I like this proposal. Not sure about the "Shared Info" card, but we can tackle this in #106 separately :slightly_smiling_face:
